packages/main/src/SliderBase.js

```diff
@@ -39,7 +39,7 @@ const metadata = {
 		},
 		/**
 		 * Defines the size of the slider's selection intervals. (e.g. min = 0, max = 10, step = 5 would result in possible selection of the values 0, 5, 10).
-		 * If 0 no visible interval between value changes will appear. When negative number, the component fallbacks to its default value.
+		 * If set to 0 the slider handle movement is disabled. When negative number or value other than a number, the component fallbacks to its default value.
```

Comment:
I did not get this "0" handling. Should not we handle it the same way as negative numbers. If someone wants to have disabled slider, it can use the disabled property.